lindeck-frontend/src/views/boody/loginPage/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,58 @@
+import TextField from "@material-ui/core/TextField";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+import React, {useRef, useState} from "react";
+import Button from "@material-ui/core/Button";
+import {makeStyles} from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import {navigate} from "@reach/router";
+import GS from "../../../../common/classes/GlobalStorage";
+
+
+const useStyles = makeStyles((theme) => ({
+    submit: {
+        margin: theme.spacing(3, 0, 2),
+    }
+}))
+
+export default function LoginForm(props) {
+    const refToEmailField = useRef(null);
+    const refToPasswordField = useRef(null);
+
+    const classes = useStyles();
+
+    const [loginGood, setLoginGood] = useState("");
+
+    let myFields = [
```

Comment:
Pleease don't use "my" in variable names

Reply:
/srug
 removed all "My", and  setted to "Our"  !!!  (ok just removed)